docs/user/dashboard/lens-end-to-end/lens-end-to-end.asciidoc

```diff
@@ -0,0 +1,442 @@
+
+[float]
+[[histogram]]
+=== Monitor your website traffic per hour 
+
+A histogram chart
+
+Do you want to add a new post with the intention of reaching as many users as possible? 
+Or do you need to make a change in your website that requires a short outage of your service? In both cases knowing an hourly website traffic distribution will help. 
+
+. From the dashboard, click *Create panel*, then click *Lens* on the *New visualization* window.
+
+. Make sure the *kibana_sample_data_logs* index appears.
+
+. From the *Available fields* list, drag and drop *bytes* to *Vertical axis*.
+
+. From the editor, click *Bytes*.
+
+.. From *Select a function*, click *Sum*.
+
+.. In the *Display name* field, enter `Transferred bytes`.
+
+.. From the *Value format* dropdown, select `Bytes (1024)`.
++
+[role="screenshot"]
+image::images/lens_end_to_end_5_1.png[Vertical axis configuration]
+
+. From the *Available fields* list, drag and drop *hour_of_day* to *Horizontal axis*.
+
+. From the editor, click *hour_of_day*, then modify a slider *Intervals granularity* to get hourly intervals.
++
+[role="screenshot"]
+image::images/lens_end_to_end_5_2.png[Create custom ranges]
+
+. From the toolbar, click *Save*.
+
+.. In the *Title* field, enter `Hourly Traffic Distribution`. 
+
+.. Select *Add to dashboard after saving*.
+
+.. Click *Save and return*.
+
+[float]
+[[custom-ranges]]
+===  Visualize the percentage of transfered file sizes 
+ 
+To compare the amount of small versus large transfered files on your webpage, create a pie chart.
+
+. From the dashboard, click *Create panel*, then click *Lens* on the *New visualization* window.
+
+. Make sure the *kibana_sample_data_logs* index appears.
+
+. From the *Available fields* list, drag and drop *bytes* to *Vertical axis* in the editor.
+
+. From the editor, click *Average of bytes*.
+
+.. From *Select a function*, click *Sum*.
+
+.. Click *Close*.
+
+. From the *Available fields* list, drag and drop *bytes* to *Break down by* in the editor.
+
+. From the editor, click *bytes*.
+
+.. From the *Value format* dropdown, select *Bytes (1024)*.
+
+.. Click *Create custom ranges*, enter `0 -> 10240` in the *Ranges* field, then 
+enter `Below 10KB` in the *Label* field. 
+
+.. Click *Add range*, enter `10240 -> +∞` in the *Ranges* field, the enter `Above 10KB`in the *Label* field.
++
+[role="screenshot"]
+image::images/lens_end_to_end_6_1.png[Custom ranges configuration]
++
+TIP: You can also use the *Filters* function to achieve the same results with <<kuery-query,KQL>> and <<lucene-query,Lucene>> filters.
+
+. From the *Chart Type* dropdown, select *Pie*.
++
+[role="screenshot"]
+image::images/lens_end_to_end_6_2.png[Files size distribution]
+
+. From the toolbar, click *Save*.
+
+.. In the *Title* field, enter `File size distribution`. 
+
+.. Select *Add to dashboard after saving*.
+
+.. Click *Save and return*.
+
+[float]
+[[treemap]]
+===  Visualize the top locations of your website traffic
```

Comment:
Hmmm, the chart is more about where traffic comes from (twitter, facebook and the rest) in three top locations... I think this title can be confusing. How about 'Visualize sources of traffic in top locations'